### ISSUE TITLE
Docks are growing when selecting nodes/ways

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1318]
**[Submitted to the original trac issue database at 9.40am, Thursday, 6th November 2008]**

My layout of the MainWindow is as follows: On the left I have the Layers dock and the Properties dock, in the middle the map and on the right the Info dock and the Undo dock.

With this layout I have the weird behaviour that every way or node that I select causes the left panel (with the Layers and Properties dock) to grow by half a centimeter leaving less room for the map. The panel cannot be made smaller by dragging the (right) border (it can be made bigger though). I guess that the minimum size of one of the docks is increased each time you select a way/node, but since I'm not a C++ expert this is just my guess.

This behaviour started after the templatized tags. Before, I had not have the problem.

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 10.44am, Thursday, 6th November 2008]**

Strange, I don't see this behaviour.

Could you state your OS/Qt version, please.

Also, screenshots would be nice.

**Comment by tomhughes** [comment]
**Author: richardbrinkman@hotmail.com**
**[Added to the original trac issue at 12.23pm, Thursday, 6th November 2008]**

qt-4.4.3-1.fc9.x86_64 (Fedora 9)
Made with /usr/bin/qmake-qt4 -unix TRANSDIR_MERKAARTOR=/home/brinkman/usr/share/merkaartor TRANSDIR_SYSTEM=/usr/share/qt4/translations OSMARENDER=1 NODEBUG=1 NOWEBKIT=1GEOIMAGE=1-o Makefile Merkaartor.pro

I uploaded 3 screenshots:
before.jpg: before any road was selected
afterfirstselection.jpg: after I selected a single road
aftertenselections.jpg: after I selected 10 roads (one at a time)

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 11.19am, Saturday, 8th November 2008]**

I can reproduce on Linux but not on Windows. QT issue?

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 4.50pm, Saturday, 8th November 2008]**

Actually, the problem seems to exists on 0.12, too, before the templates.

I'm baffled that nobody reported this before...

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 10.21pm, Saturday, 8th November 2008]**

It looks like the problem is specific to the "plastique" Qt style, which is the default on Linux, on recent Qt version.

Executing merkaartor with, e.g., the "cleanlooks" style, do not exhibit the problem.

To do so, run "merkaartor -style cleanlooks".

We can just hope the problem will be solved on future releases of Qt.

**Comment by tomhughes** [comment]
**Author: aboaboit**
**[Added to the original trac issue at 5.33pm, Sunday, 13th September 2009]**

Looked at the QT buglist and saw nothing applicable, so I submitted a new entry (with additional info pointing back here)

**Comment by tomhughes** [comment]
**Author: aboaboit**
**[Added to the original trac issue at 6.28pm, Sunday, 13th September 2009]**

Replying to [comment:5 koying]:
> It looks like the problem is specific to the "plastique" Qt style, which is the default on Linux, on recent Qt version.
> 
> Executing merkaartor with, e.g., the "cleanlooks" style, do not exhibit the problem.

0.14 here and I can reproduce the problem even with cleanlooks style: select a region, then click a road outside that region. From that point on, it goes on growing.

**Comment by tomhughes** [comment]
**Author: aboaboit**
**[Added to the original trac issue at 6.33pm, Sunday, 13th September 2009]**

Forgot to say, I had "custom qt style: oxygen" ticked. It doesn't happen without.

**Comment by tomhughes** [comment]
**Author: Koying**
**[Added to the original trac issue at 11.51am, Tuesday, 15th September 2009]**

Could you state your Qt version, please.

As said before, this is a Qt version specific bug (< 4.5.1, I think)

**Comment by tomhughes** [comment]
**Author: aboaboit**
**[Added to the original trac issue at 8.03pm, Tuesday, 15th September 2009]**

Replying to [comment:9 Koying]:
> Could you state your Qt version, please.
> As said before, this is a Qt version specific bug (< 4.5.1, I think)

You're right: the first test (when I wrote the report) was with 4.5.0. Weird part was that having a custom style exposed the bug. Tried again with 4.5.2 from QT SDK 2009.03 and I can't reproduce it anymore. 

Case close, sorry for the noise and congrats for the great program :)
